src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -501,6 +512,41 @@ public function getXEditableChoices(FieldDescriptionInterface $fieldDescription)
         return $locale;
     }
 
+    /**
+     * @param string|array $role
+     * @param object|null  $object
+     * @param string|null  $field
+     *
+     * @return bool
+     */
+    public function isGrantedAffirmative($role, $object = null, $field = null)
+    {
+        if (null === $this->securityChecker) {
+            return false;
+        }
+
+        if (null !== $field) {
+            $object = new FieldVote($object, $field);
+        }
+
+        if (!is_array($role)) {
+            $role = [$role];
+        }
+
+        foreach ($role as $oneRole) {
+            try {
+                $grant = $this->securityChecker->isGranted($oneRole, $object);
```

Comment:
```php
<?php
if ($this->securityChecker->isGranted($oneRole, $object)) {
    return true;
}
```

Reply:
No, but a `return false;` in the catch block, right @greg0ire ?

Reply:
Ah no wait, return false is not possible at this point 😄 🤔